ci_default_post_tendrl.yml

```diff
@@ -22,3 +22,10 @@
       yum:
         name: ansible
         state: latest
+      register: ansible_update_status
+
+    - name: restart tendrl services
```

Comment:
Could you add "WORKAROUND" prefix to a task name? Just to make this clear later in ansible log.

Reply:
Good point! I have it updated, but not pushed here...